js/controller.js

```diff
@@ -2406,7 +2414,8 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
       }
     },
 
-    setClosedCaptionsLanguage: function() {
+    setClosedCaptionsLanguage: function(params) {
```

Comment:
Since you're modifying the function signature, can you add a jsdoc for this?

Reply:
Added